FLT/DedekindDomain/FiniteAdeleRing/BaseChange.lean

```diff
@@ -224,21 +227,103 @@ noncomputable def adicCompletionComapAlgHom (w : HeightOneSpectrum B) :
       apply UniformSpace.Completion.extensionHom_coe
     rw [this, ← IsScalarTower.algebraMap_apply K L]
 
+lemma v_adicCompletionComapAlgHom
```

Comment:
This is a weird name. It's on main now, with a weird name, because it might already be annoying enough to merge main. The global version is called `IsDedekindDomain.HeightOneSpectrum.valuation_comap`.